images/win/scripts/Installers/Validate-SQLExpress.ps1

```diff
@@ -0,0 +1,15 @@
+$sqlConn = New-Object System.Data.SqlClient.SqlConnection
+$sqlConn.ConnectionString = "Server=$env:computername\SQL2019;Integrated Security=false;User ID=sa; Password=P@ssword!!"
+$sqlConn.Open()
+$str1=""
+$sqlConn.State
+IF (Compare-Object $sqlConn.State $str1)
+{
+Write-Host "Failed attempt"
+exit 1
+}
+ELSE
+{
+Write-Host "Success"
+exit 0
+}
```

Comment:
Create a simple function to test connection, good example - https://mcpmag.com/articles/2018/12/10/test-sql-connection-with-powershell.aspx

Reply:
We have to add this software to the readme, please take a look at another installation\validation scripts

Reply:
Thank you. Function has been implemented in the code. please review.